auto-update-pip-packages/README.md

```diff
@@ -2,18 +2,12 @@
 
 ## Auto-update pip packages
 
-### Install
-
-Install the cron job. As root:
+This script is triggered by Continuous Integration of OpenFisca-France and OpenFisca-Web-API repositories (see their `.travis.yml` respective files).
```

Comment:
There are two scripts in this package. Make [This script]() point towards the actual script, to avoid any ambiguity.